Source/Fuse.Gestures/Internal/TwoFinger.uno

```diff
@@ -77,17 +75,16 @@ namespace Fuse.Gestures.Internal
 		public event TwoFingerRotateHandler Rotated;
 		public event TwoFingerTranslateHandler Translated;
 		
-		void OnLostCapture()
+		void IGesture.OnLostCapture(bool forced)
 		{
 			_point[0].Down = _point[1].Down = -1;
-			Pointer.ReleaseCapture(this);
 			if (_begun)
 			{
-				_node.EndInteraction(this);
 				if (Ended != null)
 					Ended();
 			}
 			_begun = false;
+			_trackingKeyboard = false;
```

Comment:
Should this be wrapped in a try-finally block, so we know that `_trackingKeyboard` becomes `false` even if an `Ended`-handler throws?

Thinking about it a bit more, events kinda feel a bit crappy for this, as one event-handler throwing is going to ruin the fun for all the others, leaving them in a limbo-state... Why does programming have to be so awful?